src/Middleware/Diagnostics/Diagnostics.slnf

```diff
@@ -0,0 +1,45 @@
+{
+  "solution": {
+    "path": "C:\\Code\\aspnet\\AspNetCore\\src\\Middleware\\Middleware.sln",
```

Comment:
Make relative

Reply:
Oof, good catch.

Reply:
It's unfortunate that the default VS experience for this doesn't use relative paths, even though they are supported. I may file a VS feedback item for that ;).